tapir/coop/templates/coop/membership_agreement_pdf.default.html

```diff
@@ -0,0 +1,259 @@
+{% load static %}
+{% load django_bootstrap5 %}
+{% load accounts %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Beteiligungserklärung {{ owner.first_name }} {{ owner.last_name }}</title>
+    <link rel="stylesheet" href="{% static "coop/css/membership_agreement_pdf.css" %}">
+</head>
+<body>
+
+<!-- Falzmarken -->
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 60mm;"></span>
+<span style="border-top: 1px solid grey; width: 5mm; position: fixed; left: -15mm; top: 165mm;"></span>
+
+<!-- A DIN 5008 Letter has 2,7cm of top margin -->
```

Comment:
If we're having multiple of these now, maybe it makes sense to have a DIN 5008 letter in the core and have `{% block %}` placeholders to fill in the content?

No blocker, feel free to merge like this, we'll clean it up later.

Reply:
Yes, absolutely!